src/frontend/hooks/useAuthState.ts

```diff
@@ -0,0 +1,4 @@
+import * as React from "react";
```

Comment:
Eventually the navigation will just need access to auth state, so abstraction will simplify that